Unable to install libpstat on x86_64

```
$ crew install libpstat
libpstat: Library for getting information about running processes
https://github.com/jcnelson/libpstat
version 030f37
No precompiled binary available for your platform, downloading source...
######################################################################## 100.0%
Archive downloaded
Unpacking archive, this may take awhile...
gcc (GCC) 4.9.4
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
cc -Wall -g -fPIC -std=c99 -o "/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/tools/pstat.o" -I. -I/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build -I/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/include/pstat -c "tools/pstat.c" -D_THREAD_SAFE -D__STDC_FORMAT_MACROS -D_LINUX
cc -Wall -g -fPIC -std=c99 -o "/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/libpstat.o" -I. -I/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build -I/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/include/pstat -c "libpstat.c" -D_THREAD_SAFE -D__STDC_FORMAT_MACROS -D_LINUX
cc -Wall -g -fPIC -std=c99 -o "/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/os/common.o" -I. -I/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build -I/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/include/pstat -c "os/common.c" -D_THREAD_SAFE -D__STDC_FORMAT_MACROS -D_LINUX
cc -Wall -g -fPIC -std=c99 -o "/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/os/linux.o" -I. -I/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build -I/usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/include/pstat -c "os/linux.c" -D_THREAD_SAFE -D__STDC_FORMAT_MACROS -D_LINUX
In file included from tools/pstat.h:24:0,
                 from tools/pstat.c:22:
./libpstat.h:26:19: fatal error: stdio.h: No such file or directory
 #include <stdio.h>
                   ^
compilation terminated.
In file included from libpstat.c:22:0:
libpstat.h:26:19: fatal error: stdio.h: No such file or directory
 #include <stdio.h>
                   ^
compilation terminated.
make: *** [Makefile:131: /usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/libpstat.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [Makefile:131: /usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/tools/pstat.o] Error 1
In file included from os/common.h:24:0,
                 from os/common.c:22:
./libpstat.h:26:19: fatal error: stdio.h: No such file or directory
 #include <stdio.h>
                   ^
compilation terminated.
make: *** [Makefile:131: /usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/os/common.o] Error 1
In file included from os/common.h:24:0,
                 from os/linux.h:26,
                 from os/linux.c:25:
./libpstat.h:26:19: fatal error: stdio.h: No such file or directory
 #include <stdio.h>
                   ^
compilation terminated.
make: *** [Makefile:131: /usr/local/tmp/crew/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz.dir/libpstat-030f37a9d22ef3b8be839d487c97759e6126c30f/build/os/linux.o] Error 1
libpstat failed to install: `make -j8` exited with 2
```

--- a/packages/libpstat.rb
+++ b/packages/libpstat.rb
@@ -3,12 +3,12 @@ require 'package'
 class Libpstat < Package
   description 'Library for getting information about running processes'
   homepage 'https://github.com/jcnelson/libpstat'
-  version '030f37'
+  version '030f37-1'
   source_url 'https://github.com/jcnelson/libpstat/archive/030f37a9d22ef3b8be839d487c97759e6126c30f.tar.gz'
   source_sha256 '71e5d3f190fe045b38f31dfa9b5962fb6ea4099f46b25981b411122569ff6b30'
 
   def self.build
-    system "make"
+    system "make -I #{CREW_PREFIX}/include"
   end
 
   def self.install

--- a/packages/libpstat.rb
+++ b/packages/libpstat.rb
@@ -8,6 +8,8 @@ class Libpstat < Package
   source_sha256 '71e5d3f190fe045b38f31dfa9b5962fb6ea4099f46b25981b411122569ff6b30'
 
   def self.build
+    system "sed -i 's,PREFIX ?= /usr/local,PREFIX ?= #{CREW_PREFIX},' Makefile"
+    system "sed -i 's,LIBDIR ?= $(DESTDIR)/$(PREFIX)/lib,LIBDIR ?= $(DESTDIR)/#{CREW_LIB_PREFIX},' Makefile"
     system "make -I #{CREW_PREFIX}/include"
   end
 

